include/fmt/format.h

```diff
@@ -2260,7 +2260,7 @@ FMT_CONSTEXPR20 auto write(OutputIt out, T value,
     precision = 1;
   }
   if (const_check(std::is_same<T, float>())) fspecs.binary32 = true;
-  if (!std::numeric_limits<T>::is_iec559 || std::numeric_limits<T>::digits > 64)
+  if (const_check(!std::numeric_limits<T>::is_iec559) || std::numeric_limits<T>::digits > 64)
```

Comment:
Maybe:
```c++
if (const_check(!std::numeric_limits<T>::is_iec559 || std::numeric_limits<T>::digits > 64))
```
?

``std::numeric_limits<T>::digits`` is constexpr too.

Reply:
and please apply clang-format

Reply:
Fixed